pom.xml

```diff
@@ -455,6 +455,7 @@
           <reuseForks>false</reuseForks>
           <threadCount>1</threadCount>
           <parallel>suites</parallel>
+          <perCoreThreadCount>false</perCoreThreadCount>
```

Comment:
What is this one good for?

Reply:
Passing tests :-)
Otherwise tests might be executed in parallel within same JVM which some tests don't like.